src/main/java/de/tudarmstadt/ukp/jwktl/parser/util/ITemplateParameterHandler.java

```diff
@@ -0,0 +1,66 @@
+/*******************************************************************************
+ * Copyright 2013
+ * Ubiquitous Knowledge Processing (UKP) Lab
+ * Technische Universität Darmstadt
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+package de.tudarmstadt.ukp.jwktl.parser.util;
+
+import de.tudarmstadt.ukp.jwktl.api.entry.WiktionaryWordForm;
+
+/**
+ * Implementation of this interface are used to handle parameters of templates.
+ * 
+ * @author Alexey Valikov
+ */
+public interface ITemplateParameterHandler {

```

Comment:
This interface is particularly designed for `WiktionaryWordForm`s, although it is not indicated by its name. Thus, the interface is not very generic and can hardly be reused. I suggest removing the interface or revising it such that other components can reuse it effectively.

Reply:
I see your point but not quite sure how to address this.
I want to keep separate handler classes, so I need some common interface for them.
This is, indeed, specific to `WiktionaryWordForm`s. I'm not quite sure how to allow other components to reuse it effectively. At the moment, there's neither need nor use case for that. The interface is also quite simple and will hardly be refactoring-resistant in the future.

Suggestion: just rename it to `IWiktionaryWordFormTemplateParameterHandler` to make the name reflect the specifics.

Reply:
Agree with renaming it

Reply:
Done.